project/redis-master.sls

```diff
@@ -0,0 +1,17 @@
+# Install Redis, listening on all interfaces.
+# This does not open the firewall for Redis, so do that elsewhere
```

Comment:
Do any current projects need to share a Redis server from multiple hosts?  If not, we could just keep this simple and listen on localhost only, and not complicate it until we have to.